web/cmd/directremotecontrol/main.go

```diff
@@ -53,7 +53,7 @@ type Arguments struct {
 	Port   utils.NetPortFlag `flag:"port,default=8080"`
 }
 
```

Comment:
These `rdk` loggers were colliding with the `rdk` logger in `entrypoint.go` since they were named the same, but since these seem to be mostly responsible with logging issues with rdk startup (and there already exists a `startup`), I thought `entrypoint` would be more fitting